js/animator.js

```diff
@@ -148,6 +148,60 @@ var animator = animator || {};
       this.frameWebps.push(promise);
     }
 
+    loadFromFile(file) {
+      let imageCanvas = document.createElement('canvas');
+      imageCanvas.width = this.w;
+      imageCanvas.height = this.h;
+      var reader = new FileReader();
+      var img = new Image();
+      let thumbnailWidth = 96;
+      let thumbnailHeight = 72;
+      let thumbnailContainer = document.getElementById('thumbnail-container');
+      reader.readAsDataURL(file)
+      reader.onload = (evt) => {
```

Comment:
Please add 'onerror' handling; refer to webpToVP8 in webm.js for an example.

Reply:
Added basic error handling